addons/website_forum/static/src/js/website_forum.share.js

```diff
@@ -46,17 +48,25 @@ var ForumShare = publicWidget.registry.socialShare.extend({
      * @private
      */
     _render: function () {
-        var $question = this.$('article.question');
+        var questionEl = this.el.querySelector("article.question");
+        const parser = new DOMParser();
         if (!this.targetType) {
             this._super.apply(this, arguments);
         } else if (this.targetType === 'social-alert') {
-            $question.before(qweb.render('website.social_alert', {medias: this.socialList}));
+            this.socialAlertEl = parser.parseFromString(
```

Comment:
Don't use this, it's ugly :wink: You can just add the element using insertAdjacentHTML then get the added node afterwards if required.

Reply:
@qsm-odoo Done, Thanks.

Reply:
@msh-odoo @qsm-odoo Security alert here (JS003)

Reply:
The destroy does not seem adapted at all so it does not seem done...
For the security alert, once you are sure it's good, ping the security team about it and explain why it's safe :+1:

Reply:
@qsm-odoo It seems that the destroy method is adapted, doesn't it? Or I don't get it.

Reply:
Where is `socialAlertEl` defined?